packages/ag-charts-community/src/sparklines-util.ts

```diff
@@ -5,6 +5,7 @@ export * from './util/json';
 export * from './util/angle';
 export * from './util/zip';
 export * from './util/number';
+export * from './util/deprecation';
```

Comment:
Could we expose this from `module-support.ts` rather than this entry-point (the intended scope of this entry-point was historically the Grid Sparklines module)?